_includes/specials/bech32/23-compat.md

```diff
@@ -1,4 +1,4 @@
-As of this writing, here are what we think are some of the most
+{% auto_anchor %}As of this writing, here are what we think are some of the most
```

Comment:
Note: the lack of newline here is deliberate.  I did a weird thing when adding this to [Newsletter 60](https://bitcoinops.org/en/newsletters/2019/08/21/#bech32-sending-support) and so we can't have a newline here.